packages/release-utils/src/utils.ts

```diff
@@ -1,11 +1,16 @@
-import { getPackages, Package } from "@manypkg/get-packages";
-// @ts-ignore
-import mdastToString from "mdast-util-to-string";
+import { getPackages, type Package } from "@manypkg/get-packages";
+import cp, { ChildProcess } from "node:child_process";
```

Comment:
```suggestion
import cp, { type ChildProcess } from "node:child_process";
```

This might need a `type` here (IIUC following the rest of the code patterns)

Reply:
Good catch. It's not strictly required by tests right now but we want to maintain this style. With TS 5.8 (soon to be released) we'll be able to configure `erasableSyntaxOnly` and then the CI will validate that the code has correct imports, exports and more